modules/tinymce/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Added
 - New `sidebar_show` option to show the specified sidebar on initialization #TINY-8710
+- New `newline_behavior` option to control whether blocks or linebreaks are inserted #TINY-8458
```

Comment:
Maybe something like this it might not be clear that this applies to the enter key action only.

```suggestion
- New `newline_behavior` option to control whether blocks or linebreaks are inserted when pressing the enter key #TINY-8458
```

Reply:
The problem here is that it controls how all newlines are inserted pretty much, such as via the `mceInsertNewLine` command.

Reply:
Yes but I agree some sort of mention of the enter key would be helpful.